server/templates/staff/course/enrollment.html

```diff
@@ -178,3 +166,28 @@ <h3 class="box-title">Add/Edit Participant</h3>
 
                 <!-- </body> do not close body in template-->
                 {% endblock %}
+
+{% block page_js %}
+<script>
+  $(document).ready(function() {
```

Comment:
You can remove the `$(document).ready` wrapper. I added that only because list.js wasn't loaded at the time this script was run, but it should be now.